frontend/src/host_orchestrator/orchestrator/cvd/cvd.go

```diff
@@ -87,11 +88,10 @@ func (c *Command) Run() error {
 }
 
 func cvdEnv(androidHostOut string) []string {
-	env := []string{}
-	if androidHostOut != "" {
-		env = append(env, envVarAndroidHostOut+"="+androidHostOut)
+	if androidHostOut == "" {
+		return nil
 	}
-	return env
+	return append(os.Environ(), envVarAndroidHostOut+"="+androidHostOut)
```

Comment:
nit: I think it would be useful having the explanation written in the commit message as a note here.

Reply:
Done